public/locales/ru-RU.json

```diff
@@ -380,7 +380,7 @@
   "Opening Block": "Открывающий блок",
   "Closing Block": "Закрывающий блок",
   "Total Value Locked (TVL)": "Общая заблокированная стоимость (TVL)",
-  "Automatic restaking": "Автоматический повторный стейкинг",
```

Comment:
This term was invented and is not very intuitive. I changed it on someting any one can understand without looking for explanation